dev/run-python-flavor-tests.sh

```diff
@@ -29,5 +29,6 @@ pytest tests/utils/test_model_utils.py --large
 pytest tests/tracking/fluent/test_fluent_autolog.py --large
 pytest tests/autologging --large
 find tests/spark_autologging/ml -name 'test*.py' | xargs -L 1 pytest --large
+python dev/test_mlflow_lazily_imports_pyspark.py
```

Comment:
pytest executes import statements in test scripts when collecting testing functions. Some of our test scripts contain `import pyspark`. As a workaround, run `test_mlflow_lazily_imports_pyspark.py` in a separate process.

Reply:
Could you add this comment into the script file ?

Reply:
got it